doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -256,7 +256,10 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`bt_fast_pair_readme` service:
+
+  * Added the :c:func:`bt_fast_pair_info_cb_register` function and the :c:struct:`bt_fast_pair_info_cb` structure to register Fast Pair information callbacks.
+    The new :c:member:`bt_fast_pair_info_cb.account_key_written` can be used to notify the application about the Account Key writes.
```

Comment:
The :c:member:`bt_fast_pair_info_cb.account_key_written` callback can be used to notify the application about the Account Key writes.